src/main/kotlin/co/yappuworld/team/client/application/AdminTeamService.kt

```
@@ -0,0 +1,240 @@
+        activityUnitIds: List<UUID>
+    ) {
+        val teamMembers = activityUnitIds.map { activityUnitId ->
+            val activityUnit = activityUnitFindService.findActivityUnit(activityUnitId)
+                ?: throw BusinessException(TeamError.INVALID_ACTIVITY_UNIT)
+
+            TeamMemberEntity(team = team, activityUnit = activityUnit)
+        }
+
+        try {
+            teamMemberCommandService.saveAll(teamMembers)
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)